tasks/postgres-cluster.yml

```diff
@@ -82,9 +83,18 @@
 
 - name: Upload recovery.conf
   template: src=recovery.conf.j2 dest=/var/lib/postgresql/{{ postgres_version }}/{{ postgres_cluster_name }}/recovery.conf owner=postgres group=postgres mode=0600
-  when: postgres_primary
+  when: postgres_primary and postgres_version | version('12', '<')
   no_log: True
 
+- name: Define secondary nodes as secondaries with corresponding signal files
+  when: postgres_primary and postgres_version | version('12', '>=')
+  file:
+    path: {{ item }}
+    state: touch
+  with_items:
+    - /var/lib/postgresql/{{ postgres_version }}/{{ postgres_cluster_name }}/standby.signal
+    - /var/lib/postgresql/{{ postgres_version }}/{{ postgres_cluster_name }}/recovery.signal
```

Comment:
I think we should keep only standby.signal, as we don't want the recovery mode.

Reply:
indeed, thanks I removed it :+1: